index.html

```diff
@@ -41,18 +41,17 @@
             <h2>NT</h2>
          </a>
          <nav>
-               <a href="#" class="mobile-button">
-                  <i class="fa-solid fa-bars"></i>
-               </a>
+               <div class="mobile-button"></div>
+               

```

Comment:
- [ ] Please make sure that users cannot scroll the page while the mobile `navbar` is open, currently I can scroll the body of the page. You can set `body { overflow: hidden}` while `navbar` is oven, this will prevent users from scrolling. 
- [ ] You have setup linters for JS, props on that 👍🏻👍🏻  Please make sure that you have no linter errors. All checks should be green 🙈 
- [ ] Lastly, please make sure that the design matches with the design provided in Figma, 

Your design  | Figma design
------------- | -------------
<img width="319" alt="Screenshot 2022-03-01 at 16 24 03" src="https://user-images.githubusercontent.com/77038610/156168588-a6c0d745-3b42-47da-9893-8e7417f637e6.png"> | <img width="344" alt="Screenshot 2022-03-01 at 16 26 26" src="https://user-images.githubusercontent.com/77038610/156168934-f73e144c-de92-4d69-85c0-ec3a0c90efda.png">

More specifically, currently, the menu items look too congested, I recommend to consider giving more margin or padding between the items.